lsp_types_sublime_text_33.py

```diff
@@ -1115,7 +1115,7 @@ class TokenFormat(Enum):
     # of a document.
     'range': NotRequired[Union[bool, dict]],
     # Server supports providing semantic tokens for a full document.
-    'full': NotRequired[Union[bool, '__Full_Type_12']],
+    'full': NotRequired[Union[bool, '__SemanticTokensOptions_full_Type_1']],
```

Comment:
hmm, this doesn't look right

I would expect `__SemanticTokensRegistrationOptions_full_Type_1` instead of `__SemanticTokensOptions_full_Type_1`

Reply:
will look into this

Reply:
Ok, got it, 
the SemanticTokensRegistrationOptions extends SemanticTokensOptions
```jsonc
{
	"name": "SemanticTokensRegistrationOptions",
	"properties": [],
	"extends": [
		// ...
		{
			"kind": "reference",
			"name": "SemanticTokensOptions"
		}
	],
	// ...
}
```
and SemanticTokensOptions has these properties, legend, range, full.
```jsonc
{
	"name": "SemanticTokensOptions",
	"properties": [
		{
			"name": "legend"
			//..
		},
		{
			"name": "range"
			//..
		},
		{
			"name": "full"
			//..
		}
	],
	"mixins": [
		//...
	],
}
```
So that is the reason why I see `__SemanticTokensOptions_full_Type_1` for the `full` property